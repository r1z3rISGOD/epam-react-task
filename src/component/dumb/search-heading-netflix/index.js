import { SearchHeadingNetflix } from './search-heading-netflix'
export { SearchHeadingNetflix }
