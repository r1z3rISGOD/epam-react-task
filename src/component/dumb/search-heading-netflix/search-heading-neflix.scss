@import "./src/_variables/variables.scss";

.search__heading-1 {
  font-size: 1.5rem;
  margin: 0;
  color: $color;
  font-family: $font;
}

.film-heading {
  display: flex;
  justify-content: space-between;
  align-items: center;
  padding-top: 20px;
  padding-bottom: 50px;
}
