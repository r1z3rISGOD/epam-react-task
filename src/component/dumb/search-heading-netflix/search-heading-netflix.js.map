{"version":3,"file":"search-heading-netflix.js","sourceRoot":"","sources":["search-heading-netflix.tsx"],"names":[],"mappings":";;;AAAA,6BAA8B;AAC9B,wCAAqC;AAM9B,IAAM,oBAAoB,GAAyC,UAAC,EAAY;QAAV,QAAQ,cAAA;IACnF,OAAO,CACH,6BAAK,SAAS,EAAC,cAAc;QACvB,4BAAI,SAAS,EAAC,mBAAmB,sBAAqB;QACvD,QAAQ,CACP,CACT,CAAA;AACH,CAAC,CAAA;AAPY,QAAA,oBAAoB,wBAOhC"}