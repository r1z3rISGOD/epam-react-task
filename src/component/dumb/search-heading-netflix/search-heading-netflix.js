import React from 'react'
import './search-heading-neflix.scss'

export const SearchHeadingNetflix = (props) => {
  return (
      <div className='film-heading'>
            <h2 className="search__heading-1">netflixroulette</h2>
          {props.children}
      </div>
  )
}
