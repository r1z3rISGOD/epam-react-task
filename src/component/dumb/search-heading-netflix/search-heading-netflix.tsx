import * as React from 'react'
import './search-heading-neflix.scss'

interface SearchHeadingNetflixProps {
    children?: React.ReactNode
}

export const SearchHeadingNetflix : React.FC<SearchHeadingNetflixProps> = ({ children }) => {
  return (
      <div className='film-heading'>
            <h2 className="search__heading-1">netflixroulette</h2>
          {children}
      </div>
  )
}
