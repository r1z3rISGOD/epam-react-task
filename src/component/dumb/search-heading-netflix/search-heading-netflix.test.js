import React from 'react'
import { SearchHeadingNetflix } from './search-heading-netflix.tsx'

it('search heading netflix', () => {
  const comp = shallow(<SearchHeadingNetflix/>)
  expect(comp).toMatchSnapshot()
})
