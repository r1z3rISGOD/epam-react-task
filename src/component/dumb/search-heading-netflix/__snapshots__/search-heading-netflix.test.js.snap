// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search heading netflix 1`] = `
<div
  className="film-heading"
>
  <h2
    className="search__heading-1"
  >
    netflixroulette
  </h2>
</div>
`;
